cordova/yarn.lock

```diff
@@ -714,6 +719,11 @@ cordova-plugin-whitelist@^1.3.4:
   resolved "https://registry.yarnpkg.com/cordova-plugin-whitelist/-/cordova-plugin-whitelist-1.3.4.tgz#31938545c7c3e7de35c20ab08c2c3afa06e8a3f9"
   integrity sha512-EYC5eQFVkoYXq39l7tYKE6lEjHJ04mvTmKXxGL7quHLdFPfJMNzru/UYpn92AOfpl3PQaZmou78C7EgmFOwFQQ==
 
+cordova-plugin-zeroconf@^1.4.2:
```

Comment:
Reviewer note - I noticed that Cordova was adding this automatically when running `cordova plugin add ...` locally - so I assumed it was missed out in the past and decided to commit it as well.